107/src/test/resources/ehcache-107-default-copiers.xml

```diff
@@ -35,4 +40,29 @@
     <ehcache:heap>10</ehcache:heap>
   </ehcache:cache-template>
 
+  <ehcache:cache-template name="stringtemplate">
```

Comment:
Its reused because it already talks about default copiers, and that's the change here.